charts/infra-apps/values.yaml

```diff
@@ -133,7 +133,7 @@ kubePrometheusStack:
   # -- Chart
   chart: kube-prometheus-stack
   # -- [kube-prometheus-stack Helm chart](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack) version
-  targetRevision: 42.1.0
+  targetRevision: 44.2.1
```

Comment:
Do we need to take care of that when updating or is this handled on update? https://artifacthub.io/packages/helm/prometheus-community/kube-prometheus-stack#from-43-x-to-44-x (update the CRDs), which is in their documentation on what to do when jumping from 43. to 44. Same applies for the jump from 42. -> 43